src/chia_plot.cpp

```diff
@@ -365,6 +370,14 @@ int main(int argc, char** argv)
 		std::cout << "Invalid finaldir: " << final_dir << " (needs trailing '/' or '\\')" << std::endl;
 		return -2;
 	}
+	if(!stage_dir.empty() && stage_dir.find_last_of("/\\") != stage_dir.size() - 1) {
+		std::cout << "Invalid stagedir: " << stage_dir << " (needs trailing '/' or '\\')" << std::endl;
+		return -2;
+	}
+	if(!stage_dir.empty() && tmptoggle) {
```

Comment:
This check needs to be above line `stage_dir = tmp_dir;`

Reply:
Moved it above the assignment. Apologies for so many issues, it worked well in my testing. But I apparently didn't check for everything. I appreciate your reviews.

Reply:
Found trailing / check also needed to be moved up.

Reply:
yeah it's normal ;)